runbotics/runbotics-orchestrator-ui/src/src-landing/components/BlogCard/BlogCard.tsx

```diff
@@ -43,6 +43,8 @@ const BlogCard: FC<BlogCardProps> = ({ post, className }) => {
                             fill
                             alt={post.imageAlt ?? ''}
                             className={styles.img}
+                            sizes='(max-width: 1920px) 20vw'
+                            placeholder='empty'
```

Comment:
what does this change provide?

Reply:
Those are optional for this issue but recommended by Next. 'Sizes' provides information about how wide the image will be at different breakpoints. It is used by the browser to determine which size of the image to download, from next/image's automatically-generated source set. 'Placeholder' is to determine the state when image is loading.